tpp-app/src/onboarding/Welcome.js

```diff
@@ -1,42 +1,49 @@
 import React from 'react';
-import {StyleSheet, Text, Image, ImageBackground, TouchableOpacity} from 'react-native';
+import {StyleSheet, Text, Image, ImageBackground, View } from 'react-native';
 import MNationIcon from '../../ios/tppapp/Images.xcassets/AppIcon.appiconset/1024.png'
 import OnboardingBackground from '../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/background.png'
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import PeriodStart from './PeriodStart';
 import PeriodLength from './PeriodLength';
 import SymptomsChoices from './SymptomsChoices';
-import { MyTabs as MainPage } from '../../App';
+import Backup from './Backup';
+import Registration from './Registration'
+import Password from './Password';
+import { MainPage } from '../../App';
+import Success from './Success';
+import Confirmation from './Confirmation';
+import { WideButton } from './components/ButtonComponents';
 
 export const STACK_SCREENS = {
```

Comment:
what's the point of this if the keys & values are same? Is it so it's explicit (via the variable name) that these are screens? Not really opposed, just confused

Reply:
I took Roy's suggestion from my last ticket, this is mostly for avoiding spelling mistakes when navigating to the pages in the future.

Reply:
yeah that makes sense